src/connector/src/with_options.rs

```diff
@@ -100,6 +101,10 @@ pub trait WithPropertiesExt: Get {
         connector.contains("-cdc")
     }
 
+    fn is_shared_cdc_source(&self) -> bool {
+        self.is_cdc_connector() && CdcTableType::from_properties(self).can_backfill()
```

Comment:
Can you elaborate? This is just extracted from the current implementation in `create_source`. What is should be like?

Reply:
Yeah, because we do this check in `create_source`, so the context is to create a Source. But if without this context, we cannot distinguish it. `is_shared_cdc_source` in the trait can be called in other place.

Reply:
Fair enough. 

BTW, what are the other cases? When don't we create shared CDC source for `mysql-cdc`? (Do you mean `CREATE TABLE` directly? Any other cases?)

And the original question: Can we `CREATE SOURCE` for `citus-cdc`? If not, where is it rejected?

Reply:
I renamed the function to `is_backfillable_cdc_connector`. I think there won't be any confusion now.

For `BoundSource`, we can call its method `is_shared_cdc_source`

Reply:
> For BoundSource, we can call its method is_shared_cdc_source

Seems not true. `CREATE TABLE`  will also have a Source node.. Let me just call it `is_backfillable_cdc_connector`